client/ionic/ios/App/App/capacitor.config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "appId": "int.who.app",
```

Comment:
noting that this might change in the future for consistency with Android, where `int` is a reserved word